nj/conf.py

```diff
@@ -0,0 +1,15 @@
+"""conf"""
```

Comment:
Perhaps this should be a yaml config dotfile in the user's home dir instead of burying the config in the project structure..?